src/NHSD.BuyingCatalogue.Organisations.Api/Settings/RegistrationSettings.cs

```diff
@@ -0,0 +1,16 @@
+﻿using NHSD.BuyingCatalogue.Organisations.Api.Services;
+
+namespace NHSD.BuyingCatalogue.Organisations.Api.Settings
+{
+    /// <summary>
+    /// Registration settings.
+    /// </summary>
+    internal sealed class RegistrationSettings
```

Comment:
Do we need a `RegistrationSettings` class if it only contains the email message. Could EmailMessage be a parameter on the RegistrationService instead? It would reduce the amount of code & we don't know yet whether we will need anymore settings, unless you have some insight?